source/frontend/src/components/common/mapFSM/useLocationFeatureLoader.tsx

```diff
@@ -63,6 +68,11 @@ const useLocationFeatureLoader = () => {
           | undefined = undefined;
 
         // Load PimsProperties
+        if (selectedFeature?.pimsLocationFeature?.PROPERTY_ID) {
+          pimsLocationProperties = await loadProperties({
+            PROPERTY_ID: selectedFeature.pimsLocationFeature.PROPERTY_ID,
+          });
+        }
         if (parcelFeature !== undefined) {
```

Comment:
this call might need to be in the else clause of the previous if, otherwise it risks re-assigning the pimsLocationProperties value